static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -56,6 +91,17 @@ define([
                 });
             }
 
+            var answersEmailSignupForms = $('.js-storyquestion-email-signup-form');
+
+             if (answersEmailSignupForms) {
+                answersEmailSignupForms.each(function (el) {
+                    bean.on(el, 'submit', function(event) {
+                        submitSignUpForm(event, el)
```

Comment:
You don't need to pass the element, it will be available as `event.currentTarget` and so you can simplify here:
`el.addEventListener('submit', submitSignupForm)`